a_file

```diff
@@ -0,0 +1 @@
+If you see a robbery at an Apple Store does that make you an iWitness?
```

Comment:
I'm not gonna lie, this took me a while, but I chuckled. :D

Reply:
Please add a comment explaining why the joke is funny. It isn't obviously clear.

Reply:
PTAL

Reply:
Nothing has changed from what I can see.

Reply:
Yeah, me either.